x-pack/plugin/shutdown/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/shutdown/NodeShutdownIT.java

```diff
@@ -235,6 +241,15 @@ public void testStalledShardMigrationProperlyDetected() throws Exception {
         });
     }
 
+    /**
+     * Ensures that attempting to delete the status of a node that is not registered for shutdown gives a 404 response code.
+     */
+    public void testDeleteNodeNotRegisteredForShutdown() throws Exception {
+        Request deleteReq = new Request("DELETE", "_nodes/this-node-doesnt-exist/shutdown");
```

Comment:
This needs the assumeTrue:

```suggestion
        assumeTrue("must be on a snapshot build of ES to run in order for the feature flag to be set", Build.CURRENT.isSnapshot());
        Request deleteReq = new Request("DELETE", "_nodes/this-node-doesnt-exist/shutdown");
```

Reply:
Nice catch, thanks!